src/you_get/extractors/youku.py

```diff
@@ -18,14 +18,18 @@ def fetch_cna():
                 log.i('Found cna in imported cookies. Use it')
                 return cookie.value
     url = 'http://gm.mmstat.com/yt/ykcomment.play.commentInit?cna='
-    req = urllib.request.urlopen(url)
-    return req.info()['Set-Cookie'].split(';')[0].split('=')[1]
+    while True:
```

Comment:
while True:
...     cna = fetch_cna()
...     c += 1
...     print(cna, c)
...     if not '+' in cna:
...         break

I get c == 1000+, without break.

if I set cna to const string, it fine.